### ISSUE TITLE
About  3DETR Finetuning on SUNRGBD

### ISSUE THREAD

**Issue by wangyePHD** [BLVLab/PiMAE#3]
Thanks for your motivated work. I just want to know the details about the 3DETR finetuning configuration on SUNRGBD. The default finetuning epoch is 1080, but the the epoch of training from the scratch is also 1080 in 3DETR paper. Therefore,  Why train 1080 epoch after pre-training on sunrgb-d dataset?

**Comment by antonioo-c** [comment]
Thanks! The reasons for training 1080 epoch after pre-training on SUNRGB-D dataset are listed below.

1. We use the default training epoch for finetuning to demonstrate the effectiveness of PiMAE under fair comparison (i.e. only appying our pretrained weights & lower learning rate)
2. Previous pretraining methods in 2D/3D (such as MultiMAE, PointContrast, CrossPoint), were also comparing their pretrained model with scratch model under the same finetuning epoch.

Hope this answers your question. Have a good day.
